guacamole/src/main/frontend/package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "private": true,
+    "scripts": {
+        "build": "webpack --progress"
+    },
+    "dependencies": {
+        "@simonwep/pickr": "1.8.1",
+        "angular": "1.8.2",
+        "angular-route": "1.8.2",
+        "angular-templatecache-webpack-plugin": "^1.0.1",
```

Comment:
Do we plan on having any policy on whether these dependencies should use the `^` operator or not? The `package-lock.json` ensures that builds will be repeatable, but I usually like to lock down the packages to specific versions by removing the `^` and other operators out of `package.json` as well.

I'm not really sure what the right call is here.

Reply:
Reading up on this a bit, it seems the best approach is to use `^` except where there's a specific reason not to:

* It's the default for npm and therefore the least surprising to anyone already well-steeped in that ecosystem.
* It avoids duplicating the duty of `package-lock.json`, which we are including in version control here following the _other_ established best practice for npm.

I definitely prefer the Maven approach of absolute reproducibility by default (with the Maven equivalent to `^` and `~` being considered bad practice and almost never used), but we're stuck with the npm approach here.

I'll update to 🥕s everywhere.